### DISCUSSION TITLE
Keep copyright information

### DISCUSSION THREAD

**Discussion by BenjaminHoegh** [tjko/jpegoptim#88] [General]
Add option to keep copyright information on a image when optimize it.

This would be handy to use when creating a online image gallery for photographer so there images don't lose the copyright information but still get optimized so it fast to load

**Comment by tjko** [comment]
Existing command line option (--strip-none) should work for this?

**Reply by BenjaminHoegh** [reply]
will this still remove other types of metadata?

**Reply by tjko** [reply]
With _--strip-none_ it will keep all markers (metadata).

By default all common metadata (EXIF, XMP, ICC, IPTC, and COMments) are kept, other (unknown) data is discarded...
